aries-site/src/pages/templates/scrolling-and-pagination.mdx

```diff
@@ -0,0 +1,3 @@
+import { ComingSoon } from '../../components';
+
+<ComingSoon />
```

Comment:
Looks Good! 
I think we need to update our coming soon page but this isn't relevant to this PR